### DISCUSSION TITLE
[Question]  What's the difference between Fluss and ForSt DB?

### DISCUSSION THREAD

**Discussion by yfcck** [apache/fluss#46] [Q&A] [answered]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Fluss version

0.5.0

### Minimal reproduce step

No response

### What doesn't meet your expectations?

No response

### Anything else?

Could you please specifically explain the use cases and differences between Fluss and Forst DB?

Fluss currently supports Flink 1.20. Is there any internal plan to align with Flink 2.0?

Can Fluss 0.5 support the upcoming release of Paimon 1.0?

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by polyzos** [comment]
Hi @yfcck 👋 I moved this as a question instead of bug.
Forst DB is a state store used in Flink 2.0 to store state... Fluss is a real-time streaming storage, it's a separate system and it will support the upcoming release of Paimon 1.0

**Comment by aib628** [comment]
> Hi @yfcck 👋 I moved this as a question instead of bug. Forst DB is a state store used in Flink 2.0 to store state... Fluss is a real-time streaming storage, it's a separate system and it will support the upcoming release of Paimon 1.0

Would it be more appropriate to use the discussions module to explore this issue?

**Comment by wuchong** [comment]
@yfcck I have migrated the issue to the discussion. And please use English in the future, thank you~

**Reply by yfcck** [reply]
ok ！ ths

**Comment by wuchong** [comment, marked as answer]
@yfcck Forst DB is a statebackend storage for Flink. Fluss is a table storage for the source and sink result of Flink. 

Yes, Fluss will integrate with Flink 2.0 and Paimon 1.0 when they are released.
